Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/common/AbstractSourceLayout.java

```diff
@@ -152,4 +152,21 @@ public boolean isOptional() {
     public abstract void setValue(String value);
     
     public abstract boolean validate();
+    
+    /**
+     * Activates all fields of an optional input, in a relaunched simulation context.
+     * Avoids implementation in subtypes which are not concerned.
+     * 
+     */
+    public void enableInputRelaunchedSimulation() {
+    }
+    
+    /**
+     * If true, the input value can be modifiable by loadInputs(...) method of LaunchFormLayout.java.
+     * This is in a relaunched simulation context.
+     */
+    public boolean isModifiableRelaunchedSimulation() {

```

Comment:
Not the better method name in my opinion "isValueModifiable" or "isValue Conflictual" seem better

Reply:
Yes I agree. In the last commit, I propose "isValueModifiable".